src/ameba/formatter/todo_formatter.cr

```diff
@@ -22,6 +22,7 @@ module Ameba::Formatter
         return
       end
 
+      issues.sort_by!(&.rule.name)
```

Comment:
I'd also update the `rule_todo` method to sort the issues by filename.

Reply:
I'd add another issue to the test case with different filename to ensure the ordering is kept though.